pkg/common/utils/utils.go

```diff
@@ -373,10 +373,17 @@ func GetPlaceholderFlagFromPodSpec(pod *v1.Pod) bool {
 		}
 	}
 
-	if value := GetPodLabelValue(pod, constants.LabelPlaceholderFlag); value != "" {
+	if value := GetPodAnnotationValue(pod, constants.OldAnnotationPlaceholderFlag); value != "" { // nolint:staticcheck
 		if v, err := strconv.ParseBool(value); err == nil {
 			return v
 		}
 	}
+
+	if value := GetPodLabelValue(pod, constants.OldLabelPlaceholderFlag); value != "" { // nolint:staticcheck
+		if v, err := strconv.ParseBool(value); err == nil {
```

Comment:
nit: add a WARN log message about the deprecation

Reply:
Done.